src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_help.py

```diff
@@ -514,12 +521,12 @@
 
 helps['storage share exists'] = """
     type: command
-    short-summary: Returns a boolean indicating whether the share exists.
+    short-summary: Determine if a share exists.
 """
 
 helps['storage share list'] = """
     type: command
-    short-summary: List file shares in a storage account.
+    short-summary: List file shares.
```

Comment:
This should say "in a storage account" since they are contained within.

Reply:
This is one thing that I encountered a lot that I'm not sure how to handle: References of the style where the command category is mentioned in the summary. It should be clear from context what is being listed, but then the description essentially becomes the command name. I'm interested in @JasonRShaver's thoughts on how often we want to be repeating information in CLI help, but I don't see a problem with being more specific.

Reply:
Well, one might think "List file shares" could list all file shares in all storage accounts in a subscription. That's not the case here.